aip/general/0180.md

```diff
@@ -139,6 +139,30 @@ code, as such changes will be seen as breaking by those users.
 whether a proposed change is likely to break users, and an expansive reading of
 this guidance could ostensibly prevent _any_ change (which is not the intent).
 
+#### Field behavior compatibility
```

Comment:
I'm thinking that this actually belongs in AIP-203, while AIP-180 should link to there. WDYT?

I'm thinking about maintainability primarily: if we put all field behavior discussion into one AIP, it's easy to find that and change it.

Reply:
Sure let me do that at the end though once we have all comments resolved

Reply:
SGTM! still LGTM on my end, let me know when it's ready for review again.

Reply:
I've moved it now!